tests/terraform/checks/resource/aws/example_IAMStarResourcePolicyDocument/main.tf

```
@@ -35,6 +35,25 @@ resource "aws_iam_policy" "pass" {
   })
 }
 
+resource "aws_iam_policy" "pass_unrestrictable" {
+  name        = "test_policy"
+  path        = "/"
+  description = "My test policy"
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = [
+          "ec2:ListAllMyBuckets",
```

```suggestion
          "s3:ListAllMyBuckets",
```

@gruebel I think this should be `s3:ListAllMyBuckets`, since `ec2:ListAllMyBuckets` is not a valid action.

---

😄 thanks!